analysis/alignment/requantAlignedValues.py

```diff
@@ -447,7 +447,6 @@ def runSingleFileImputation(options, peakgroups_file, mzML_file, method, is_test
     rid = list(swath_chromatograms.getRunIDs())[0]
 
     start = time.time()
-    initial_alignment_cutoff = 0.0001
```

Comment:
note: the new pyprophet default algorithm does not produce low q-values any more...

Reply:
This is due to the empirical p-values, where the most confident p-values are bound by the number of queries in the library. If we use a library containing only hundreds or thousands, that might be an issue. On the other hand, the estimates seem to be more accurate than before, so we might just increase this value. Even better might be to use PEP, as this is more directly linked to the quality of peak groups.

Reply:
now the user can set this parameter him/herself and we should be fine. before it was hardcoded